use_cases.py

```diff
@@ -8,13 +8,13 @@
 # from aws.src.database.multi_table.amend_tenure import main as amend_tenure_main
 # from aws.src.database.multi_table.find_persons_for_properties import main as find_persons_for_properties_main
 
-from aws.src.database.dynamodb.scripts.asset_table.patch_asset_with_additional_boiler_house_id import main as update_boilerhouse
+from aws.src.elastic_search.housing_search.update_properties import main as update_properties
 
 if __name__ == "__main__":
     # 1) IMPORTANT: Uncomment and click into functions in the imports to view definitions and edit config
     #     - e.g. set STAGE to "development" or "production"
     # 2) Call functions here
 
-    update_boilerhouse()
+    update_properties()
```

Comment:
I would leave this function commented out, if eventually this gets pushed to master. 
Whoever uses this repo should change this file before running anything